share/spack/spackos-bootstrap/stage1/spack.in.yaml

```diff
@@ -0,0 +1,19 @@
+# This is a Spack Environment file.
+#
+# It describes a set of packages to be installed, along with
+# configuration settings.
+spack:
+  include:
+    - compilers.yaml
```

Comment:
Where is this include file supposed to be found? I feel like the spackos-bootstrap directory is missing a `Makefile` or something similar :thinking: